src/core/generators/mutator.ts

```diff
@@ -47,21 +48,18 @@ export const generateMutator = async ({
     ? 'ErrorType'
     : `${pascal(name)}ErrorType`;
 
-  const { file, cached } = await loadFile<Record<string, Function>>(
-    importPath,
-    {
-      isDefault: false,
-      root: workspace,
-      alias: mutator.alias,
-      tsconfig,
-    },
-  );
+  const { file, cached } = await loadFile<string>(importPath, {
+    isDefault: false,
+    root: workspace,
+    alias: mutator.alias,
+    tsconfig,
+    load: false,
+  });
 
   if (file) {
-    const mutatorFn =
-      file[mutator.default || !mutator.name ? 'default' : mutator.name];
+    const mutatorInfo = parseFile(file, isDefault ? 'default' : mutator.name!);
```

Comment:
Looks like the condition `isDefault ? 'default' : mutator.name!` can be extracted into a separate variable and reused [at line 66](https://github.com/anymaniax/orval/pull/325/files#diff-3bf59b28833123acbdd18b785a76b03fe4e417ad1426e394d89cdd93d9dad23dR66)

Reply:
done